src/test/java/org/project/SpotifyEndToEndTests.java

```diff
@@ -19,14 +19,14 @@ public void setUpDriver() {
         uiTest.setUp();
     }
 
-    @AfterMethod
-    public void closeDriver() {
-        uiTest.close();
-    }
+  //  @AfterMethod
```

Comment:
uncomment this code, because after test run 4-5 opened browser windows stay unclosed.